### PR TITLE
Login with JWT

### PR DESCRIPTION
**Pull request by DarthHater**
This is a branch based off of the Material UI work that does a quick lil login with JWT. It doesn't do anything with state really yet, just sets a token and proves that https://github.com/DarthHater/bored-board-service/pull/13 works!

### CODE REVIEW

File: src/components/Login/Login.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+                throw(error);
+            });
+    }
+
+    onUserNameChange(event) {
+        this.setState({username: event.target.value});
+    }
+
+    onPasswordChange(event) {
+        this.setState({password: event.target.value});
```
Comment (conwaydev) [inline src/components/Login/Login.js]: I dont know if we want to save the password to state, we should probably just submit it with the form and not keep it in JS at all

Comment (conwaydev) [reply src/components/Login/Login.js]: like in `handleSubmit` we serialize the form and pass the values to AuthService

File: src/components/NavigationBar/NavigationBar.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, {Component} from 'react';
+import {Link} from 'react-router-dom';
+import AppBar from 'material-ui/AppBar';
+import FlatButton from 'material-ui/FlatButton';
+
+class NavigationBar extends Component {
+    
+    constructor() {
```
Comment (conwaydev) [inline src/components/NavigationBar/NavigationBar.js]: we dont actually need a constructor if its not being used

**Review by conwaydev — APPROVED** [review]

File: src/components/Login/Login.js (hunk 1 above)
Comment (DarthHater) [reply src/components/Login/Login.js]: In: 2203a0a

File: src/components/NavigationBar/NavigationBar.js (hunk 2 above)
Comment (DarthHater) [reply src/components/NavigationBar/NavigationBar.js]: Old habits or whatever. I'll nix it.

**Comment by DarthHater** [comment]
I've still got a few things to work out here @conwaydev , mainly getting a redirect to login working, and then making the login form actually interactive. If you want to jump in on any of it go for it.

File: src/components/NavigationBar/NavigationBar.js (hunk 2 above)
Comment (DarthHater) [reply src/components/NavigationBar/NavigationBar.js]: In: e34b656

**Comment by conwaydev** [comment]
<3

**Pull request merged**